cfpb/capital-framework
# Use CFPB brand colors in cf-buttons component variables

Adds CFPB brand colors as the default variable values for colors in `cf-buttons`. Our docs show the brand colors already, since they have a `cf-theme-overrides.less` stylesheet.

Depends on #561 - this includes changes from that branch in the diff.

## Testing

In this repo:
- check out this branch
- run `npm run build`
- run `npm run cf-link`

In a second local clone of this repo in a different folder:

- pull down and check out the `gh-pages-rm-overrides` branch: `git fetch && git checkout gh-pages-rm-overrides`
- if you haven't already in this second clone, [run all the install steps for the project](https://github.com/cfpb/capital-framework/tree/gh-pages#installation)
- run `npm run cf-link`
- run `npm run build`
- run `npm start`
- Compare colors on local site: 
   - http://localhost:3000/components/cf-buttons/ 
   - to the live cf documentation to make sure they match: https://cfpb.github.io/capital-framework/components/cf-buttons/

## Review

- @user

## Screenshots
![screen shot 2017-08-17 at 11 12 15 am](https://user-images.githubusercontent.com/702526/29419408-11e6be40-833d-11e7-8e15-11e359317185.png)


## Notes

-

## Todos

-

## Checklist
